habitat_baselines/il/trainers/eqa_cnn_pretrain_trainer.py

```diff
@@ -0,0 +1,264 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import time
+
+import numpy as np
+import torch
+from torch.utils.data import DataLoader
+
+import habitat
+from habitat import logger
+from habitat_baselines.common.base_il_trainer import BaseILTrainer
+from habitat_baselines.common.baseline_registry import baseline_registry
+from habitat_baselines.common.tensorboard_utils import TensorboardWriter
+from habitat_baselines.il.data.eqa_cnn_pretrain_data import (
```

Comment:
Seems like the dataset didn't get committed

Reply:
Ya thanks, it got ignored because all `data` folders were specified in .gitignore. I have added it now.